src/codegeneration/CommonJsModuleTransformer.js

```diff
@@ -129,10 +129,11 @@ export class CommonJsModuleTransformer extends ModuleTransformer {
       let descriptor;
 
       switch (exp.type) {
-        case GET_ACCESSOR:
+        case GET_ACCESSOR: {
```

Comment:
Oh, I didn't realize the commit description was referring to Traceur source rather than output.